src/scripts/syncBreaches.js

```diff
@@ -99,7 +99,6 @@ if (seen.size !== breaches.length) {
 
   // get
   const result = await getAllBreaches()
-  console.log(result.length)
```

Comment:
We'd want to keep this console log. It's good to know in the logs how many total breaches we currently have

Reply:
9ce1d2fd9